src/enums/routeNames.ts

```diff
@@ -34,4 +34,11 @@ export enum RouteNames {
   RESTORATION_BUSINESS_INFORMATION = 'restoration-business-information',
   RESTORATION_BUSINESS_NAME = 'restoration-business-name',
   RESTORATION_REVIEW_CONFIRM = 'restoration-review-confirm',
+
+  // Amalgamation (regular) route names
+  AMALG_REG_BUSINESS_INFO = 'amalg-reg-business-info',
+  AMALG_REG_INFORMATION = 'amalg-reg-information',
+  AMALG_REG_PEOPLE_ROLES = 'amalg-reg-people-roles',
+  AMALG_REG_REVIEW_CONFIRM = 'amalg-reg-review-confirm',
+  AMALG_REG_SHARE_STRUCTURE = 'amalg-reg-share-structure',
```

Comment:
I named these so (and also the views) because I expect that horiz/vert short-form amalgamations will be separate -- they have 4 steps instead of 5 and different validations, etc.